frontend/next.config.js

```diff
@@ -12,7 +12,9 @@ module.exports = withSourceMaps({
     conferenceCode: process.env.CONFERENCE_CODE || "pycon-demo",
     SENTRY_DSN: process.env.SENTRY_DSN || null,
   },
-
+  images: {
+    domains: ['production-pycon-backend-media.s3.amazonaws.com'],
+  },
```

Comment:
shall we add pycon.it as well?

Reply:
for assets coming from nextjs we shouldn't need to add the domain as the URL will be /XX

what use case are you thinking where we will have an URL like `pycon.it/media/image.png`?